nest-city-account/Dockerfile

```
@@ -18,6 +21,10 @@ RUN npm ci --include=dev --frozen-lockfile \
  && npx prisma generate
 
 FROM build-base AS build
+WORKDIR /root/openapi-clients
+COPY --from=openapi-clients /app/ ./
```

(optional) You shouldn't need to change `WORKDIR` before copying. You probably can just do 
```suggestion
COPY --from=harbor.bratislava.sk/standalone/openapi-clients:${SHARED_PACKAGE_TAG} /app/ /root/openapi-clients/
```